src/platform/plugins/shared/controls/public/controls/data_controls/options_list_control/options_list_strings.ts

```diff
@@ -8,14 +8,14 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import type { OptionsListSearchTechnique } from '../../../../common/options_list/suggestions_searching';
+import type { OptionsListSearchTechnique } from '@kbn/controls-schemas';
 import { MAX_OPTIONS_LIST_BULK_SELECT_SIZE } from './constants';
 
 export const OptionsListStrings = {
   control: {
     getDisplayName: () =>
       i18n.translate('controls.optionsList.displayName', {
-        defaultMessage: 'Options list',
+        defaultMessage: 'options list',
```

Comment:
This fixes the edit action title, since we are now using the normal panel action rather than a custom control edit action:

<img width="658" height="162" alt="image" src="https://github.com/user-attachments/assets/2db1a485-7082-49e6-8546-da418c37ea01" />